opm/parser/eclipse/EclipseState/Tables/PlymaxTable.hpp

```diff
@@ -34,11 +34,14 @@ namespace Opm {
          * \brief Read the PLYMAX keyword and provide some convenience
          *        methods for it.
          */
-        void init(Opm::DeckItemConstPtr item)
+        void init(Opm::DeckRecordConstPtr record)
         {
-            SimpleTable::init(item,
-                              std::vector<std::string>{"C_POLYMER", "C_POLYMER_MAX"});
-
+            createColumns(std::vector<std::string>{"C_POLYMER", "C_POLYMER_MAX"});
```

Comment:
do you think the names are a little misleading? I think for PLYMAX is for maximum polymer concentration and maximum salinity.